boards/arm/olimex_stm32_h103/olimex_stm32_h103.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: olimex_stm32_h103
+name: OLIMEX-STM32-H103
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 20
+flash: 128
+supported:
+  - gpio
```

Comment:
All supported peripherals should be listed here (you can take other boards as reference). If I'm not wrong, this enables more tests.

Reply:
Does that mean all supported peripherals by the MCU, or this port? I only tested a few

Reply:
I assume all the ones enabled in the dts file are supported, so they should be listed here.